### ISSUE TITLE
SourceBans Checker to include comms blocks

### ISSUE THREAD

**Issue by sneak-it** [sbpp/sourcebans-pp#188]
Just a bit of a suggestion/enhancement/feature request, it would be nice if the SourceBans checker plugin also counted & announced how many comms blocks a connecting user has, in similar fashion to previous # of bans being listed.

**Comment by Technoblazed** [comment]
I wrote something for this a while back, no idea where the code went though \o/. Also implemented mute evasion into sourcesleuth

**Comment by sneak-it** [comment]
@Technoblazed If you ever run across it and would like to share, I'm sure several people would be interested.

**Comment by Technoblazed** [comment]
I'll take a look for it this weekend :)

**Comment by sarabveer** [comment]
@Technoblazed if you ever find it, re-implement it with the latest version of SourceSleuth and submit a PR.

**Comment by DataWorm** [comment]
Seems he didn't find it or forgot about it. Anyway I also missed that feature so I just checked the code and modified it today and in the end successfully tested it with some help. sm_listcomms is now implemented but we might try to make a few more improvements. Going to make a merge request when I am done. 
Also I should mention that I never programmed on source scripts before so sorry if there is a little but code duplication which I preferred to keep it simple and avoid additional bugs.
